### PR TITLE
Various review updates

### PR DESCRIPTION
**Pull request by robertusr**

### CODE REVIEW

File: docs/spec.html [hunk 1]
```diff
@@ -2062,13 +2063,6 @@ <h3>Changes</h3>
   },{
     name: "David Hyland-Wood",
     company: "PegaSys",
-  },{
```
Comment (grantnoble) [inline docs/spec.html]: @robertusr for now, can you leave these in here, please. The javascript that tidies up the file for rendering into html for some reason fails when you tinker too much with the editor names, which is why I put these in here in the first place. We can sort out what is going wrong later.

**Review by grantnoble — CHANGES_REQUESTED** [review]
Leave the sample editors in please, otherwise the javascript fails.

File: docs/spec.html (hunk 1 above)
Comment (robertusr) [reply docs/spec.html]: Interesting, I tested the render, and it seemed to work fine for me.

Comment (robertusr) [reply docs/spec.html]: Will leave them in.

**Review by grantnoble — APPROVED** [review]
LGTM

**Pull request merged**